services/src/main/resources/locale/addon/Gamification_en.properties

```diff
@@ -376,7 +376,7 @@ realization.label.actionId=Action ID
 realization.label.actionLabel=Action label
 realization.label.actionType=Action type
 realization.label.program=Program ID
-realization.label.programLabel=Program label
+realization.label.programLabel=Program
```

Comment:
to be changed on crowdin at the same time

Reply:
done !